src/main.rs

```diff
@@ -318,7 +331,13 @@ fn work(args: &Args) -> Result<(), MainError> {
 
             if args.section.get(SectionFeatures::SIZE) {
                 let len = section.size();
-                println!("{}${:04x} ({}) bytes", indent, len, len);
+                println!(
+                    "{}${:04x} ({}) bytes{}",
+                    indent,
+                    len,
+                    len,
+                    plural!(len, "s")
+                );
```

Comment:
```suggestion
                println!(
                    "{indent}${len:04x} ({len}) bytes{}",
                    plural!(len, "s"),
                );
```
In the interest of compactness.

Reply:
There are lots of other `println("{}", somevar)`s; should I change them in general to `println!("{somevar}")`?

Reply:
Please do!